### PR TITLE
update 17.7.23

### PR DESCRIPTION
**Pull request by hohn**

### CODE REVIEW

File: README.md
Comment (knewbury01) [inline README.md:29]: can you add a note to the readme/amend the list of tested CLI versions above to include any other versions ? thanks!

File: build-multiple-codeql-versions.sh
Comment (knewbury01) [inline build-multiple-codeql-versions.sh:2]: maybe add a small usage sample in the comment? it takes an arg yeah - $1, what is that meant to be?

or better yet maybe add a small sample of how to use these in a new section in the README

File: sarif_cli/scan_tables.py
Comment (knewbury01) [inline sarif_cli/scan_tables.py:104]: wait, should this also be commented out?

File: sarif_cli/signature.py
Comment (knewbury01) [inline sarif_cli/signature.py]: is this relevant? if not perhaps rm

File: sarif_cli/table_joins.py
Comment (knewbury01) [inline sarif_cli/table_joins.py]: rm

**Review by knewbury01 — COMMENTED** [review]
hey @hohn thanks for the fix,

just a few suggestions, and one Ipython rm required! other than that lgtm, pls ping when the review is addressed!

File: sarif_cli/scan_tables.py
Comment (hohn) [reply sarif_cli/scan_tables.py:104]: Why?

File: sarif_cli/signature.py
Comment (hohn) [reply sarif_cli/signature.py]: that's just a whitespace change.  The comment is older, from 
`2ba9593d (Kristen Newbury 2022-12-01 `

File: sarif_cli/table_joins.py
Comment (hohn) [reply sarif_cli/table_joins.py]: done

File: sarif_cli/scan_tables.py
Comment (hohn) [reply sarif_cli/scan_tables.py:104]: Never mind.

Comment (hohn) [reply sarif_cli/scan_tables.py:104]: Looks like it's used by
```
File: sarif_cli/table_joins_CLI.py
335:47:        .merge(sf(1111), how="left", left_on='automationDetails', right_on='struct_id', validate="1:m")
```
so I'm hesitant.

File: sarif_cli/signature.py
Comment (hohn) [reply sarif_cli/signature.py]: I removed the comment.

**Comment by hohn** [comment]
Ready!

**Review by knewbury01 — APPROVED** [review]
lgtm! thanks!

**Pull request merged**
